client/packages.go

```diff
@@ -84,6 +84,11 @@ type Snap struct {
 	Tracks []string `json:"tracks,omitempty"`
 
 	Health *SnapHealth `json:"health,omitempty"`
+
+	// UserHold is the time until which the snap's refreshes are held by the user.
+	UserHold time.Time `json:"user-hold,omitempty"`
```

Comment:
this can be probably just be called Hold/hold  to match the same role of RefreshInfo.Hold